ui-testing-core/src/main/java/com/avito/android/test/espresso/action/scroll/ScrollToIfPossibleAction.kt

```diff
@@ -67,7 +67,7 @@ class ScrollToIfPossibleAction : ViewAction {
                 .withCause(
                     RuntimeException(
                         "Scrolling to view was attempted, " +
-                            "but the view is not displayed"
+                                "but the view is not displayed"
```

Comment:
Something wrong with formatting

Reply:
thanks, just copied formatting XML's from avito, and forgot to apply it